### ISSUE TITLE
Sticky item with different layout

### ISSUE THREAD

**Issue by cristin** [davideas/FlexibleAdapter#159]
I want to know if I can use this library and how to create the following list effect.

I want always the first displayed item from a list (lets call this a focused item) to have a different layout then the other items. When I scroll each item will receive focus, one by one. 

For example if I have items 1,2,3 displayed on screen, item 1 will have focus and a different layout. When I scroll down item 1 will be replaced in the focused layout with item 2 and on screen we will have: item 2 (has focus and is displayed with a different layout in top part of the list), items 3 and 4. If I scroll again item 2 will be replaced in the focused layout with item 3 and on screen we will have: item 3 (has focus and is displayed with a different layout in top part of the list), items 4 and 5. 

A solution that I thought of is to use the sticky functionality and make each list item a header without items. Any other suggestions?

**Comment by davideas** [comment]
Hi @cristin, not clear at all what you want to achieve. I don't understand your explanation. But please choose better your words, when you introduce your problem. However:

- You can create any different items with their names and specific layout as much as you desire.
- Focus term is not appropriate in the context for lists.
- You wanted to say, "when you scroll UP item 1 is replaced by item 2"?
- You can't create the sticky effect only with headers because sections have no subitems, so the first headerItem will just scroll up as any normal item because the second headerItem will push it up when they are adjacent, so there's no sense to create headers...
- What do you mean with "if you scroll AGAIN item2" ?? how much big are these items?
- The layout displayed is always attached to the item. Layout cannot be changed with the scroll but maybe with a state, so that you can change layout if item has different [custom] state...
- When sticky, a headerItem layout is just translated and don't change layout, why it should change?
